js/Source/Slider.Modify/Behavior.Slider.Modify.js

```diff
@@ -61,6 +64,13 @@ Behavior.addGlobalFilter('Slider.Modify', {
         roundAfterSnap: api.getAs(Number, 'roundAfterSnap')
       }
     );
+    if(formTarget){
+      var timer;
+      slider.addEvent('move', function(){
+        clearTimeout(timer);
+        timer = setTimeout(formTarget.submit, api.get('formSubmitDelay'));
+      });
+    }
```

Comment:
I think all of this should be a Behavior Plugin. I.e. if you have an app that has a slider that doesn't need this functionality, you don't need to include it. Plugins are pretty easy; you're handed the instance the filter created (in this case, the slider). Your options would be on the plugin (`formToSubmit`, `formSubmitDelay`). It keeps this behavior clean and single purpose and your plugin is nice and tidy. Give it a shot!